internal/jennies/typescript/rawtypes.go

```diff
@@ -351,3 +373,26 @@ func formatValue(val any) string {
 
 	return fmt.Sprintf("%#v", val)
 }
+
+func hasStructDefaults(typeDef ast.Type, defaults any) bool {
+	_, ok := defaults.(map[string]interface{})
+	return ok && typeDef.Kind == ast.KindStruct
+}
+
+func toOrderedMap(value any) *orderedmap.Map[string, interface{}] {
```

Comment:
This function is pretty much the same than the Go one, so I would definitely move it to the `orderedmap` package :D